src/common/table/table.module.scss

```diff
@@ -42,6 +41,10 @@ $transition-duration: 300ms;
     border-bottom: solid 2px $gray-500;
     margin-bottom: $rows-spacing;
     padding: units(0.5) 0;
+    position: sticky;
+    top: units(3);
+    background-color: $gray-50;
+    z-index: 1;
```

Comment:
Can we add a `map` or a `function` in a partial under `src/assets/styles/` to keep track of all the `z-index` values in the future? 

https://jonsuh.com/blog/organizing-z-index-with-sass/

Reply:
Great suggestion! Added! 👷‍♂️

Reply:
🚀